lib/views/customer_details.erb

```diff
@@ -30,7 +41,7 @@
               <label for="shipping-customer-name">Shipping Name<span class="text-danger">*</span></label>
               <input
                 <%if first_error?(:missing_shipping_customer_name)%>
```

Comment:
Leave a space between the if and the tag.

Reply:
I don't think you're meant to leave a space between <% and the ruby code.
For some reason, my editor really hated this line as well.
(Also, this wasn't changed in Emma's PR so wouldn't be part of what she needs to change)

Reply:
Yeah because I ran rubocop autocorrect a load of other stuff has come into this